test/org/sagebionetworks/bridge/services/ScheduledActivityServiceRecurringTest.java

```diff
@@ -179,32 +179,25 @@ public void persistedActivitiesAreFilteredByEndsOn() throws Exception {
         // This was demonstrated above, but by only one activity... this is a more exaggerated test
         
         // Four days...
-        DateTime endsOn = NOW.plusDays(4);
-        ScheduleContext context = getContext(NOW, DateTimeZone.UTC, endsOn);
+        ScheduleContext context = getContext(NOW, DateTimeZone.UTC, EST, 4);
         List<ScheduledActivity> activities = service.getScheduledActivities(context);
         
         // Zero days... there are fewer activities
-        endsOn = NOW.plusDays(0);
-        context = getContext(NOW, DateTimeZone.UTC, endsOn);
+        context = getContext(NOW, DateTimeZone.UTC, EST, 0);
         List<ScheduledActivity> activities2 = service.getScheduledActivities(context);
         
         assertTrue(activities2.size() < activities.size());
     }
     
-    private ScheduleContext getContextWith2DayWindow(DateTime now, DateTimeZone requestZone) {
-        return getContext(now, EST, now.withZone(requestZone).plusDays(2));
-    }
-    
-    private ScheduleContext getContext(DateTime now, DateTimeZone persistedZone, DateTime endsOn) {
+    private ScheduleContext getContext(DateTime now, DateTimeZone initialTimeZone, DateTimeZone requestTimeZone, int daysAhead) {
         return new ScheduleContext.Builder()
-            .withStudyIdentifier(study.getStudyIdentifier())
-            .withClientInfo(ClientInfo.UNKNOWN_CLIENT)
-            .withInitialTimeZone(persistedZone)
-            .withNow(now)
-            .withAccountCreatedOn(now)
-            // Setting the endsOn value to the end of the day, as we do in the controller.
-            .withEndsOn(endsOn.withHourOfDay(23).withMinuteOfHour(59).withSecondOfMinute(59))
-            .withHealthCode(testUser.getHealthCode())
-            .withUserId(testUser.getId()).build();
-    }
-}
+                .withStudyIdentifier(study.getStudyIdentifier())
+                .withClientInfo(ClientInfo.UNKNOWN_CLIENT)
+                .withInitialTimeZone(initialTimeZone)
+                .withRequestTimeZone(requestTimeZone)
+                .withNow(now.withZone(requestTimeZone))
```

Comment:
Does any of our functionality depend on the timezone of now? This change changes the timezone of now in some tests, which may change the functionality and test coverage.